ouroboros-network/src/Ouroboros/Network/TxSubmission/Inbound.hs

```diff
@@ -324,6 +329,12 @@ txSubmissionInbound _tracer maxUnacked mpReader mpWriter _version =
             bufferedTxs3 = forceElemsToWHNF $ bufferedTxs2 <>
                                (Map.fromList (zip live (repeat Nothing)))
 
+        writtenTxids <- mempoolAddTxs txsReady
```

Comment:
This call already exists 7 lines below :)

Reply:
Fixed.  Thanks!